libtransmission/torrent.cc

```diff
@@ -395,6 +396,7 @@ void torrentCallScript(tr_torrent const* tor, std::string const& script)
         { "TR_TORRENT_LABELS"sv, labels_str },
         { "TR_TORRENT_NAME"sv, tor->name() },
         { "TR_TORRENT_TRACKERS"sv, trackers_str },
+        { "TR_TORRENT_PRIORITY"sv, priority_str },
```

Comment:
_(nit)_ please sort them alphabetically

Reply:
On it, I thought they were in chronological order.